### PR TITLE
chore: use already decoded tx from cached DecodedTransactionResponse object

### PR DESCRIPTION
**Pull request by troggy**
Since #6 hooks like `useLiveRecentTransactions`, `useFilteredTransactions` and `useOlderTransactions` return transactions as `DecodedTransactionResponse` instead of `Horizon.TransactionResponse`. The former type extends the latter with `decodedTx` attribute which you can just use in your components.

```ts
export interface DecodedTransactionResponse extends Horizon.TransactionResponse {
  decodedTx: Transaction
}
```

This PR changes `TransactionList` and `SignatureRequestList` components to take advantage of this new attribute instead of decoding XDR in-place with `TransactionBuilder.fromXDR`.

### CODE REVIEW

File: src/Account/components/SignatureRequestList.tsx [hunk 1]
```diff
@@ -17,7 +17,6 @@ interface SignatureRequestListItemProps {
   onOpenTransaction?: (tx: Transaction, signatureRequest: MultisigTransactionResponse) => void
   signatureRequest: MultisigTransactionResponse
   style?: React.CSSProperties
-  testnet: boolean
```
Comment (sozidatel) [inline src/Account/components/SignatureRequestList.tsx:20]: @troggy, why did you remove testnet parameter?

Comment (troggy) [reply src/Account/components/SignatureRequestList.tsx:20]: because it is not needed anymore. Was needed for `TransactionList` props, where it was needed for `TransactionBuilder.fromXDR` call only. Now that we don't call `fromXDR` anymore, the prop is not needed neither in `TransactionList` nor in `SignatureRequestList`.

**Review by GSA05 — APPROVED** [review]
